ArjunaJTA/jta/pom.xml

```diff
@@ -272,13 +251,16 @@
         <dependency>
           <groupId>org.wildfly.arquillian</groupId>
           <artifactId>wildfly-arquillian-container-managed</artifactId>
-          <version>${version.org.wildfly.arquillian}</version>
           <scope>test</scope>
         </dependency>
         <dependency>
           <groupId>org.jboss</groupId>
+<<<<<<< HEAD
```

Comment:
@moulalis will you resolve this merge conflict please.

Reply:
> @moulalis will you resolve this merge conflict please.

@mmusgrov Good catch, I've updated the changes.